CHANGELOG.md

```diff
@@ -0,0 +1,515 @@
+
+### Changed
+* Switch to policy endpoint for job results by @cd-work (#1006)
+* Reformat `phylum history` output by @kylewillmon (#1010)
+
+### Fixed
+* Allow `phylum pip install -e .` on macOS by @kylewillmon (#1017)
+* Skip analysis with empty package list by @cd-work (#1007)
+
+### Removed
+* Remove `phylum project set-thresholds` subcommand by @cd-work (#1004)
+* Remove request type from global config by @kylewillmon (#1001)
+
+## [4.8.0] - 2023-04-04
+
+### Added
+* Add SPDX SBOM parser by @ejortega (#963)
+* Add `pip` extension as official extension by @kylewillmon (#980)
+* Use recursive lockfile search for `phylum init` by @cd-work (#979)
+
+### Fixed
+* Use `--dry-run` output for `poetry` extension by @cd-work (#957)
+* Switch default subcommand from `list` to `help` by @cd-work (#959)
+* Fix inconsistent `phylum init` whitespace by @cd-work (#964)
+* Remove `--force` option from `phylum analyze` by @kylewillmon (#966)
+* Move extension API source to extension directory by @cd-work (#969)
+* Improve sandboxed process failure message by @cd-work (#972)
+* Allow calling `phylum` from subdirectories by @matt-phylum (#974)
+* Improve lockfile parsing errors by @cd-work (#992)
+* Fix `phylum project link` overwriting project file by @cd-work (#995)
+* Add SPDX tag:value parser by @ejortega (#978)
+* Make `phylum package` type argument mandatory by @cd-work (#997)
+
+## [4.7.0] - 2023-02-27
+
+### Added
+* Add automatic lockfile detection by @cd-work (#950)
+
+### Fixed
+* Fix project history endpoint by @cd-work (#947)
+
+## [4.6.1] - 2023-02-14
+
+### Fixed
+* Fix Go parser ignoring dependencies by @cd-work (#944)
+
+## [4.6.0] - 2023-02-03
+
+### Added
+* Improve `phylum init` UX by @cd-work (#936)
+
+## [4.5.0] - 2023-02-01
+
+### Added
+* Add multi-lockfile ecosystems to analysis summary by @cd-work (#925)
+* Add option to specify multiple lockfiles on CLI by @cd-work (#927)
+
+### Fixed
+* Fix poetry extension by @cd-work (#926)
+* Fix install with poetry extension by @cd-work (#930)
+
+### Upgrading
+There are no breaking changes in this release. Projects may like to take
+advantage of the new `.phylum_project` file format which accounts for multiple
+lockfiles. To do so, simply run the `phylum init` command from the root of the
+project directory. As long as the project and group names used are the same as
+before, the existing project ID will be re-linked.
+
+## [4.4.0] - 2023-01-20
+
+### Added
+* Add `phylum group delete` subcommand by @cd-work (#916)
+* Add multi-lockfile support to `phylum init` by @cd-work (#910)
+
+### Fixed
+* Abort on unknown extension subcommands by @cd-work (#915)
+* Fix gem parser for dependencies without version by @cd-work (#919)
+
+## [4.3.0] - 2023-01-17
+
+### Added
+* Add multi-lockfile support to `.phylum_project` by @kylewillmon (#902)
+* Make config file write atomic by @cd-work (#892)
+
+### Fixed
+* Fix sandbox executable path resolution by @cd-work (#905)
+
+
```

Comment:
Are the double newlines here intentional? It doesn't match the rest of the file as far as I can tell.

Reply:
Not intentional. Good catch.